qucs/qucs/main.cpp

```diff
@@ -876,7 +895,7 @@ int main(int argc, char *argv[])
     }
     // create netlist from schematic
     if (netlist_flag) {
-      return doNetlist(inputfile, outputfile);
+      return doNetlist(inputfile, outputfile, netlang);
```

Comment:
Possibly, it is better to put "netlang" as the property of the "Schematic" class object.
Schematic::getNetlist can return the netlist then.